File: from-2201-to-2300/2270-number-of-ways-to-split-array/solution.go
func waysToSplitArray(nums []int) int {
    leftSum, rightSum := 0, 0
    for _, num := range nums {
        rightSum += num
    }
    result := 0
    for i := 0; i < len(nums) - 1; i ++ {
        leftSum += nums[i]
        rightSum -= nums[i]
        if leftSum >= rightSum {
            result += 1
        }
    }
    return result
}

File: from-2201-to-2300/2270-number-of-ways-to-split-array/solution.py
class Solution:
    def waysToSplitArray(self, nums: List[int]) -> int:
        leftSum, rightSum = 0, sum(nums)
        result = 0

        for num in nums[:-1]:
            leftSum += num
            rightSum -= num
            if leftSum >= rightSum:
                result += 1

        return result